### ISSUE TITLE
Provide a way to change user passwords

### ISSUE THREAD

**Issue by amenonsen** [Nordaaker/convos#302]
Right now, one has to generate the new password and paste it into the user's .json file. It would be helpful to have a command that takes a username and prompts for a new password instead.

(In the longer term, it would be nice for users to be able to reset their password themselves, if they have forgotten it.)

**Comment by jhthorsen** [comment]
This is the way right now:

```
killall convos;
perl -Ilib -Ilocal/lib/perl5 -MConvos::Core::User -e'print Convos::Core::User->new->set_password("supersecretpassword")->password'
$EDITOR ~/.local/share/convos/$CONVOS_EMAIL/user.json
./script/convos
```

I'm thinking this should be wrapped inside a sub command:

```
./script/convos set-password [email] <password>
```

And possibly prompt, if password is not given as input. I'm not quite sure if that can be done with the core modules though. Will have a look.

Also, would be nice if the "set-password" command forced convos to re-read the user config files, so the server did not need to restart.

**Comment by jhthorsen** [comment]
This is now implemented in #323.
